pkg/management/endpoint.go

```diff
@@ -699,6 +702,40 @@ func MakeUpdateAuthorizationsEndpoint(component Component) cs.Endpoint {
 	}
 }
 
+// MakePutAuthorizationEndpoint creates an endpoint for PutAuthorization
+func MakeAddAuthorizationEndpoint(component Component) cs.Endpoint {
+	return func(ctx context.Context, req interface{}) (interface{}, error) {
+		var m = req.(map[string]string)
+		var err error
+
+		var authorizations api.AuthorizationsRepresentation
+
+		if err = json.Unmarshal([]byte(m[reqBody]), &authorizations); err != nil {
+			return nil, errorhandler.CreateBadRequestError(msg.MsgErrInvalidParam + "." + msg.Body)
+		}
+
+		return nil, component.AddAuthorization(ctx, m[prmRealm], m[prmGroupID], authorizations)
```

Comment:
The authorization param is kind of useless because, it is specified in the partial matrix, currently I ignore this param in this endpoint. So I see two alternatives : 

- Remove this param of the api, but it seems to me right to keep it as it is the same params for GET and DELETE
- Use it, just for checking that there is a match between the partial matrix and this param